tests/monotouch-test/UIKit/SimpleTextPrintFormatterTest.cs

```diff
@@ -31,6 +31,10 @@ public void DefaultCtor ()
 			using (var stpf = new UISimpleTextPrintFormatter ()) {
 				Assert.That (stpf.Handle, Is.Not.EqualTo (IntPtr.Zero), "Handle");
 				if (TestRuntime.CheckXcodeVersion (11, 0)) {
+					Assert.NotNull (stpf.Color, "Color");
```

Comment:
I think you need to change the condition in the previous line to be Xcode 14 (and not Xcode 11)